transformer/kubernetes/apiresource/argocdapplication.go

```diff
@@ -70,6 +71,16 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	destNamespace := qaengine.FetchStringAnswer(
+		common.ConfigCICDArgoDestinationNameSpace,
+		"Enter the destination namespace for argo cd pipeline",
```

Comment:
Can we change the string to "Enter the destination namespace for the Argo CD pipeline"